packages/opentelemetry-node/test/spanprocessor.test.ts

```diff
@@ -711,6 +714,42 @@ describe('SentrySpanProcessor', () => {
       });
     });
   });
+
+  it.only('associates an error to a transaction', () => {
```

Comment:
The `only` probably should go again 😅

Reply:
thanks fixed with b9a4ac3203027935a649d7cf3475bf8037b7bfa5